### ISSUE TITLE
Bedre DIKUmemes-slide

### ISSUE THREAD

**Issue by Eckankar** [kantinen/infoscreen#30]
https://github.com/datalogisk-kantineforening/kantinfo/issues/17

> Lige nu kører jeg en DIKUmemes ting på min server: https://gist.github.com/Eckankar/e882c1235a49f44c7af0
> 
> Det fungerer fint nok, på nær, at den token vi får skal fornys med nogle få måneders mellemrum, og det sker manuelt. Er der nogen der har en bedre idé?
>

**Comment by Thraundil** [comment]
Jeg kender ikke meget til Facebook-API'et, så er lidt tom for ideer.

Er det fordi den Token man får aldrig er permanent?

**Comment by Eckankar** [comment]
Ja; problemet er, at de kræver brugerinteraktion for at fornye en token, og de varer i... 4 måneder, tror jeg det er.

**Comment by SebastianPrehn** [comment]
Jeg kunne rigtig godt tænke mig at se DIKUmemes slides på vores lille infoskærm igen.

Graph API fra Facebook er nok svaret her, men har ikke tid pt. til at kigge på det selv. Hvis nogen har lyst til at give det et skud, så eksisterer følgende dokumentation jeg lige kunne finde der virkede relevant:

https://developers.facebook.com/docs/graph-api
https://developers.facebook.com/docs/facebook-login/guides/access-tokens
https://developers.facebook.com/docs/groups-api/
